languages/en.lang

```diff
@@ -297,6 +309,7 @@ $PALANG['pAdminEdit_admin_result_success'] = 'The admin %s has been modified.';
 $PALANG['pUsersLogin_welcome'] = 'Mailbox users login to change your password and aliases.';
 $PALANG['pUsersLogin_username_incorrect'] = 'Your login is not correct. Make sure that you login with your email address!';
 $PALANG['pUsersLogin_password_incorrect'] = 'Your password is not correct!';
+$PALANG['pUsersLogin_password_recover'] = 'Did you forget your password?';
```

Comment:
clicking on a question feels a bit strange, why not putting it like this: `I forgot my password.` ?